contrib/docker/.env.prod.template

```diff
@@ -111,8 +111,10 @@ ACME_EMAIL=
 EJABBERD_DOMAIN=ejabberd.workadventure.localhost
 
 # MUST MATCH THE EJABBERD_HOST ENV
-EJABBERD_WS_URI=wss://ejabberd.workadventure.localhost:5443/ws
-EJABBERD_API_URI=http://ejabberd:5443/api
+# Protocols have to match -> wss:// is WebSocket over SSL (default) - Change to ws:// (WebSocket) if you defined EJABBERD_API_URI with http://
+EJABBERD_WS_URI=wss://ejabberd.workadventure.localhost/ws
+# Protocols have to match -> https:// is HTTP over SSL (default) - Change to http:// if you defined EJABBERD_WS_URI with ws://
+EJABBERD_API_URI=https://ejabberd.workadventure.localhost/api
```

Comment:
Looks fine, but I just realized that the EJABBERD_API_URI could be just `http://ejabberd/api` every time because it's only accessed by the other containers so it will always be used in the Docker project and not outside.

@moufmouf what do you think ?